web/profiles/joinup/src/Form/UserMultipleCancelConfirm.php

```diff
@@ -64,30 +63,40 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     $form = parent::buildForm($form, $form_state);
```

Comment:
Same, we should move the class in the right place. Also the corresponding part from `\Drupal\joinup\Routing\RouteSubscriber::alterRoutes()`.